### PR TITLE
A few linting suggestions using sphinx-lint

### PR DESCRIPTION
**Pull request by eviau-artefactual**
Addressing a few linting issues in the doc.

### CODE REVIEW

**Comment by eviau-artefactual** [comment]
This is linked to https://github.com/archivematica/Issues/issues/1640

File: admin-manual/installation-setup/customization/antivirus-admin.rst [hunk 1]
```diff
@@ -61,43 +61,43 @@ be recorded.
...
 
-	MaxFileSize=2000M
-	MaxScanSize=2000M
-	StreamMaxLength=2000M
+    MaxFileSize=2000M
+    MaxScanSize=2000M
+    StreamMaxLength=2000M
 
-	Where M is the unit Megabytes
+    Where M is the unit Megabytes
```
Comment (replaceafill) [inline admin-manual/installation-setup/customization/antivirus-admin.rst]: Should we take this line out of the literal block to make it normal text?

**Review by replaceafill — APPROVED** [review]

File: admin-manual/installation-setup/customization/antivirus-admin.rst (hunk 1 above)
Comment (eviau-artefactual) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: Hmmm good question! This is how it looks now: https://archivematica.org/en/docs/archivematica-1.15/admin-manual/installation-setup/customization/antivirus-admin/#configuration-of-clamav-values

Comment (eviau-artefactual) [reply admin-manual/installation-setup/customization/antivirus-admin.rst]: I decided that it would make sense to make it normal text :)

**Pull request merged**